lib/brakeman/options.rb

```diff
@@ -124,6 +124,11 @@ def get_options args, destructive = false
           options[:skip_libs] = true
         end
 
+        opts.on "--add-libs-path path1,path2,etc", Array, "An application relative lib directory (ex. app/mailers) to process" do |paths|
```

Comment:
Ok, I added options descriptions for the new `additional_libs_path` as well as the previously undocumented `additional_checks_path` above. I've tried to clear up the ambiguity between these path types by more clearing defining what each is for. Hopefully the "application relative lib directory" language alon with the example "app/mailers" will make it clear what this option is for.